setup.cfg

```diff
@@ -220,6 +220,7 @@ cylc.xtriggers =
     echo = cylc.flow.xtriggers.echo:echo
     wall_clock = cylc.flow.xtriggers.wall_clock:wall_clock
     workflow_state = cylc.flow.xtriggers.workflow_state:workflow_state
+    suite_state = cylc.flow.xtriggers.suite_state:suite_state
```

Comment:
Note small conflict with #5452

Reply:
(No need to do anything unless #5452 gets merged first)